charts/workflow/values.yaml

```diff
@@ -130,6 +132,15 @@ logger:
     port: "redis port"
     password: "redis password" # "" == no password
 
+fluentd:
+  syslog:
+    # Configure the following ONLY if using Fluentd to send log messages to both
+    # the Logger component and external syslog endpoint
+    # external syslog endpoint url
+    syslog_host: ""
```

Comment:
I think it should be `host` and `port` instead. `fluentd/syslog/syslog_host` is a little redundant since we're operating under the `syslog` namespace. I'm just bikeshedding at this point though, so LGTM.